spot-electron/src/logger/logger.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Implements a singleton logger.
+ */
+class Logger {
+
+    /**
+     * Logs a message on ERROR level.
+     *
+     * @param {string} message - The log message.
+     * @param {?Error} throwable - The error to be logged, if any.
+     * @returns {void}
+     */
+    error(message, throwable) {
+        this._log('ERROR', message, throwable);
+    }
+
+    /**
+     * Logs a message on INFO level.
+     *
+     * @param {string} message - The log message.
+     * @returns {void}
+     */
+    info(message) {
+        this._log('INFO', message);
+    }
+
+    /**
+     * Logs a message on WARNING level.
+     *
+     * @param {string} message - The log message.
+     * @param {?Error} throwable - The error to be logged, if any.
+     * @returns {void}
+     */
+    warning(message, throwable) {
+        this._log('WARNING', message, throwable);
+    }
+
+    /**
+     * Private function to log a message.
+     *
+     * @private
+     * @param {string} level - The log level.
+     * @param {string} message - The log message.
+     * @param {?Error} throwable - The error to be logged, if any.
+     * @returns {void}
+     */
+    _log(level, message, throwable) {
+        // eslint-disable-next-line no-console
+        console.log(`${new Date().toISOString()} - ${level} - ${message}${throwable ? ` - ${throwable}` : ''}`);
+    }
+}
+
+exports.logger = new Logger();
```

Comment:
Here are some random musings I have about logging. In the future should we be sending these logs somewhere like spot-client does? Should we store them locally? I think in the future we should also add an uncaught exception handler (I forget how I did that previously). Also, should we be using jitsi-meet-logger here too?